receiver/kafkametricsreceiver/scraper_test_helper.go

```diff
@@ -153,14 +159,50 @@ func (s *mockClusterAdmin) ListConsumerGroupOffsets(string, map[string][]int32)
 	return s.consumerGroupOffsets, nil
 }
 
+func (s *mockClusterAdmin) DescribeConfig(cr sarama.ConfigResource) ([]sarama.ConfigEntry, error) {
+	topicName := cr.Name
+	if cr.Type == sarama.BrokerResource {
+		return s.brokerConfigs, nil
+	}
+	if s.topics[topicName].ConfigEntries == nil {
+		return nil, fmt.Errorf("no config Entries found for topic")
+	}
+	configEntry := make([]sarama.ConfigEntry, 1)
+	for name, entry := range s.topics[topicName].ConfigEntries {
+		configEntry = append(configEntry, sarama.ConfigEntry{
+			Name:  name,
+			Value: *entry,
+		})
+	}
+	return configEntry, nil
+}
+
 func newMockClusterAdmin() *mockClusterAdmin {
 	clusterAdmin := new(mockClusterAdmin)
 	r := make(map[string]string)
 	r[testGroup] = testGroup
 	clusterAdmin.consumerGroups = r
 
+	strMinInsyncReplicas := strconv.Itoa(testMinInsyncReplicas)
+	strLogRetentionMs := strconv.Itoa(testLogRetentionMs)
+	strLogRetentionBytes := strconv.Itoa(testLogRetentionBytes)
+
+	brokerConfigEntry := sarama.ConfigEntry{
+		Name:  "log.retention.ms",
```

Comment:
Is this name purposefully different than what the scraper is looking for, `log.retention.hours`?